src/components/NavList.js

```diff
@@ -0,0 +1,23 @@
+import { Link, Route, Routes } from 'react-router-dom';
+import BookList from './BookList';
+import Categories from './Categories';
+
+const NavList = () => (
+  <div>
+    <nav>
+      <p>Bookstore CMS</p>
+      <ul>
+        <li>
+          <Link to="/">Books</Link>
+        </li>
+        <Link to="/Categories">Categories </Link>
+      </ul>
+    </nav>
+    <Routes>
+      <Route path="/BookList" element={<BookList />} />
+      <Route path="/Categories" element={<Categories />} />
+    </Routes>
```

Comment:
- [ **OPTIONAL** ] For best practices and to make your code be organized, I think it would be better if you move the selected code lines in your `App.js` file. It can look similar to this:
```jsx
// ./src/App.js

import React from "react"
import { Routes, Route } from "react-router-dom"

function App() {
  return (
      <NavList />
      <Routes>
        <Route path="/" element={<Books />} />
        <Route path="/categories" element={<Categories />} />
      </Routes>
  )
}
```
Your `src/components/NavList.js` file can look similar to this:
```jsx
import { Link } from 'react-router-dom';

const NavList = () => (
  <div>
    <nav>
      <p>Bookstore CMS</p>
      <ul>
        <li>
          <Link to="/">Books</Link>
        </li>
        <Link to="/Categories">Categories </Link>
      </ul>
    </nav>
  </div>
);
```
And your `src/components/BookList.js` file should be like:
```jsx 
import InputItem from './Form';

const BookList = () => {
  ...
  return (
    <>
      <div>
        <ul>
          {ListItem.map((books) => (
            <li
              key={books.id}
            >
              {books.title}
            </li>

          ))}

        </ul>
        <ul />
      </div>

      <InputItem />
    </>
  );
};
```